webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder-datatable.js

```diff
@@ -1359,7 +1359,7 @@ function reloadTable(gridId) {
 
 function createNodeTable(gridId, refresh) {
   var isResizing = false,
-    hasHandle = false,
+    hasHandle = $('#drag').length > 0,
     offsetBottom = 250;
 
   $(function () {
```

Comment:
when mutating the `hasHandle` inside the `$(function () {})` body, it goes out of scope after execution and gets its old state back, so `hasHandle` was always `false` :sweat: Damn JS...